terraform/modules/rds/security_groups.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_security_group" "rds" {
+  name        = "${var.prefix}-rds-security-group"
+  description = "RDS security group"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description     = "Allow ingress on port 5432 only from the specified security groups"
+    security_groups = var.security_group_ids
```

Comment:
We're not actually passing in `security_group_ids` here yet (this var is just passed in as `[]` for now) but will be in the near future e.g. once there's a security group for the ECS, we'll pass it in here so that ECS will be allowed "ingress" to the DB